Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -1,8 +1,13 @@
 #import "SentryOptions.h"
+#import "SentryDSN.h"
 #import "SentryError.h"
+#import "SentryEvent.h"
+#import "SentryHTTPStatusCodeRange.h"
 #import "SentryOptions+HybridSDKs.h"
+#import "SentryOptions+Private.h"
 #import "SentrySDK.h"
-#import "SentryTests-Swift.h"
```

Comment:
This stopped working for some reason, which is why i had to rewrite UrlSessionDelegateSpy in objc.

Reply:
Thanks for the info. We can fix it in another PR.